src/centralserver/admin-service/src/main/resources/openapi-definition.yaml

```diff
@@ -1,512 +1,4021 @@
+    CertificateFile:
+      description: certificate file
+      properties:
+        certificate:
+          description: the certificate file
+          format: binary
+          type: string
+      required:
+        - certificate
+      type: object
+    CertificationServiceFileAndSettings:
+      allOf:
+        - $ref: '#/components/schemas/CertificateFile'
+        - $ref: '#/components/schemas/CertificationServiceSettings'
+      description: certification service file and settings
+      type: object
+    CertificationServiceSettings:
+      description: certification service settings
+      properties:
+        certificate_profile_info:
+          description: fully qualified class name that implements the ee.ria.xroad.common.certificateprofile.CertificateProfileInfoProvider interface
+          example: ee.ria.xroad.common.certificateprofile.impl.FiVRKCertificateProfileInfoProvider
+          format: text
+          minLength: 1
+          type: string
+        tls_auth:
+          description: can the CA be used for TLS authentication
+          type: boolean
+      required:
+        - tls_auth
+        - certificate_profile_info
+      type: object
+    CodeWithDetails:
+      $ref: '../../../../../common-rest-api/src/main/resources/common-openapi-definition.yaml#/components/schemas/CodeWithDetails'
+    ConfigurationAnchor:
+      allOf:
+        - $ref: '#/components/schemas/ModificationTimestamps'
+      description: global conf anchor # potential-common-api
+      properties:
+        hash:
+          description: anchor hash
+          example: 42:34:C3:22:55:42:34:C3:22:55:42:34:C3:22:55:42:34:C3:22:55:42:34:C3:22:55:42:34:C3
+          format: hash
+          maxLength: 255
+          minLength: 1
+          type: string
+      required:
+        - hash
+        - created_at
+      type: object
+    ConfigurationPart:
+      description: configuration part
+      properties:
+        content_identifier:
+          $ref: '#/components/schemas/ConfigurationPartContentIdentifier'
+        filename:
+          example: shared-params.xml
+          format: filename
+          maxLength: 255
+          minLength: 1
+          type: string
+        optional:
+          description: if the part is optional
+          example: true
+          type: boolean
+        file_updated_at:
+          description: file updated at
+          example: '2022-01-12T00:00:00.001Z'
+          format: date-time
+          maxLength: 24
+          minLength: 24
+          type: string
+        version:
+          description: version of the conf part
+          example: 3
+          format: text
+          minLength: 1
+          type: string
+      required:
+        - content_identifier
+        - filename
+        - file_updated_at
+        - version
+        - optional
+      type: object
+    ConfigurationPartContentIdentifier:
+      description: the content identifier of a globalconf configuration part
+      example: SHARED-PARAMETERS
```

Comment:
It could also be an enumeration. Earlier it was, but I changed it in a later commit.

Content identifiers are sort of semi-dynamic construct, in principle you can define more content identifiers by adding .ini files to CS: https://github.com/nordic-institute/X-Road/blob/develop/doc/UseCases/uc-gconf_x-road_use_case_model_for_global_configuration_distribution_1.4_Y-883-8.md#224-uc-gconf_04-describe-optional-configuration-part-data

In practice there's 5 (I believe 5?) standard identifiers that are in use in typical X-Road. Some come from core modules, some from addons: https://github.com/nordic-institute/X-Road/blob/develop/src/packages/src/xroad/default-configuration/addons/center-monitoring.ini

I thought it would be better to not have fixed enumeration for the values, in case new addons are added, or someone is running their own custom addons using the mechanism, or for some weird reason configuring their own content identifiers by adding .ini configurations by hand.